docker-entrypoint.sh

```diff
@@ -18,17 +18,23 @@ PLUGIN_OPTIONS=""
 # Parse out options.
 while [ -n "$1" ]; do
   case "$1" in
-    --python-gapic-templates )
-      PLUGIN_OPTIONS="$PLUGIN_OPTIONS,python-gapic-templates=$2"
-      shift 2
-      ;;
     -- )
       shift
       break
       ;;
     * )
-      # Ignore anything we do not recognize.
-      shift
+      # If this switch begins with "--python-gapic-" or "--gapic-", then it is
+      # meant for us.
+      if [ "${1:0:15}" == "--python-gapic-" ]; then
```

Comment:
Since we use `#!/bin/bash` and not `#!/bin/sh`, let's just do 
```sh
if [[ $var == --python-gapic-* ]]; then
```
here and below (note double square brackets `[[` `]]` for conditional expression evaluation).

Reply:
Today I learned. Thanks.